test/test_jit.py

```diff
@@ -3153,6 +3153,20 @@ def constant_prop(iter):
         self.run_pass('constant_propagation', graph)
         self.assertTrue(graph.findNode("prim::Loop").outputsSize() == 2)
 
+    def test_constant_insertion(self):
+        def foo():
+            a = [1.0, 2.0, 3.0]
```

Comment:
I'd double check that the python printer prints out string lists correctly.  I am pretty sure they printout without appropriate quoting.